deploy/helm/azure-industrial-iot/Chart.yaml

```diff
@@ -15,4 +15,4 @@ maintainers:
 - name: Karapet Kostandyan
   email: kakostan@microsoft.com
 icon: https://raw.githubusercontent.com/Azure/Industrial-IoT/master/docs/media/microsoft-symbol.png
-appVersion: 2.7.105
+appVersion: 2.7.200
```

Comment:
`2.7.200` should be fine for now, but we will need to update this later on, when we know exact version that we will release.

Reply:
@marcschier, please also change the value of `image:tag` parameter in `deploy/helm/azure-industrial-iot/values.yaml` to `2.7.200`. Those two should have the same values.

Reply:
`deploy/helm/azure-industrial-iot/README.md` should also be updated to mention `2.7.200` instead of `2.7.105`. The value is mentioned there twice.

Reply:
Thanks for the thorough review - wasn't sure why it said 105, when we are 199, so assumed some post processing or so :-).